src/components/Taxonomy/TopicConnections.tsx

```diff
@@ -102,10 +109,9 @@ const TopicConnections = ({
   );
 
   return (
-    <>
-      <FieldHeader title={t("taxonomy.topics.title")} subTitle={t("taxonomy.topics.subTitle")}>
-        <HowToHelper pageId="TaxonomySubjectConnections" tooltip={t("taxonomy.topics.helpLabel")} />
-      </FieldHeader>
+    <FieldsetRoot>
+      <FieldsetLegend>{t("taxonomy.topics.title")}</FieldsetLegend>
+      <FieldsetHelper>{t("taxonomy.topics.taxonomySubjectConnections")}</FieldsetHelper>
```

Comment:
Hvorfor er dette et `Fieldset`?

Reply:
Syntes det ga mening siden flere form controls er gruppert sammen, altså emnetilknytning og å styre om det er kjerne-/tilleggsstoff, men er litt i tvil! burde det endres?

Reply:
Njaaa, tror kanskje det er litt for big-picture.